### PR TITLE
Sub admin parameter fix

### PR DESCRIPTION
**Pull request by giesler**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by deathly809** [comment]
I think it looks good, is there a test for this?

**Comment by giesler** [comment]
Tests added, including one from this fix

File: src/StackAdmin/Azs.Subscriptions.Admin/Tests/src/DirectoryTenant.Tests.ps1 [hunk 1]
```diff
@@ -97,7 +98,7 @@ InModuleScope Azs.Subscriptions.Admin {
         It "TestListDirectoryTenants" {
             $global:TestName = 'TestListDirectoryTenants'
 
-            $allDirectoryTenants = Get-AzsDirectoryTenant -ResourceGroupName System.local
```
Comment (deathly809) [inline src/StackAdmin/Azs.Subscriptions.Admin/Tests/src/DirectoryTenant.Tests.ps1]: This is why you use a variable, so you don't have to change everywhere ;)

**Comment by deathly809** [comment]
@giesler Looks good.

**Pull request merged**